modules/ROOT/nav.adoc

```diff
@@ -80,24 +75,26 @@
  ** xref:add-another-output-transform-studio-task.adoc[Add Another Output Target]
  ** xref:move-transformations-separate-file-studio-task.adoc[Move Transformations to Separate Files]
 
-* xref:eu-cloud-configuration.adoc[Configure EU Cloud in Studio]
-
-* xref:pce-configuration.adoc[Configure Private Cloud Edition in Studio]
- ** xref:pce4-configuration.adoc[Configure PCE 4 in Studio]
- ** xref:pce3-configuration.adoc[Configure PCE 3 in Studio]
- *** xref:pce-configuration-macos.adoc[MacOS]
- *** xref:pce-configuration-windows.adoc[Windows]
-
 * xref:login-platform-from-studio.adoc[]
  ** xref:set-credentials-in-studio-to.adoc[]
 
+* xref:set-anypoint-platform-cloud.adoc[Set Anypoint Platform Cloud]
+ ** xref:deploy-mule-application-task.adoc[Deploy a Mule Application to CloudHub]
+ ** xref:domain-support-concept.adoc[Domain Support]
+  *** xref:domain-studio-tasks.adoc[Design Mule Domains]
+ ** xref:eu-cloud-configuration.adoc[Configure EU Cloud in Studio]
```

Comment:
This H1 is really different in the topic...

Reply:
I'm confused. Do you mean that the configuration of EU Cloud in Studio is really different from configuring Hyperforce or the Studio domains?

Reply:
I mean the hard-coded link text is really different than the H1 in the topic: "Access the EU Control Plane from Anypoint Studio"

Reply:
Yes! I'm trying to be consistent with the titles in the nav, at least.

Reply:
OK, but it might be confusing to have the source/destination so different.